cyhy/mailer/CyhyNotificationMessage.py

```diff
@@ -58,16 +62,20 @@ class CyhyNotificationMessage(ReportMessage):
 
 <p>Cyber Hygiene scans of your host(s) conducted in the past day have detected one or both of the following:
 <ul>
-  <li>New critical and/or high vulnerabilities</li>
+  <li>New critical, high, and/or known exploited vulnerabilities</li>
   <li>New potentially risky services</li>
 </ul>
 </p>
 
-<p>{{#is_federal}}As part of <a href="https://cyber.dhs.gov/bod/19-02/">BOD 19-02</a>, critical findings need to be remediated within 15 days and high findings remediated within 30 days.{{/is_federal}}{{^is_federal}}CISA recommends remediating critical findings within 15 days and high findings within 30 days.{{/is_federal}}</p>
+<p>{{#is_federal}}As part of <a href="https://www.cisa.gov/news-events/directives/bod-19-02-vulnerability-remediation-requirements-internet-accessible-systems">BOD 19-02</a>, critical findings need to be remediated within 15 days and high findings remediated within 30 days.{{/is_federal}}{{^is_federal}}CISA recommends remediating critical findings within 15 days and high findings within 30 days.{{/is_federal}}</p>
+
+<p>{{#is_federal}}As part of <a href="https://www.cisa.gov/news-events/directives/bod-22-01-reducing-significant-risk-known-exploited-vulnerabilities">BOD 22-01</a>, any “known exploited” findings, regardless of severity, need to be remediated within two weeks.{{/is_federal}}{{^is_federal}}CISA recommends remediating known exploited vulnerabilities, regardless of severity, within two weeks.{{/is_federal}}</p>
```

Comment:
Replace smart quotes with regular quotes:
```suggestion
<p>{{#is_federal}}As part of <a href="https://www.cisa.gov/news-events/directives/bod-22-01-reducing-significant-risk-known-exploited-vulnerabilities">BOD 22-01</a>, any "known exploited" findings, regardless of severity, need to be remediated within two weeks.{{/is_federal}}{{^is_federal}}CISA recommends remediating known exploited vulnerabilities, regardless of severity, within two weeks.{{/is_federal}}</p>
```

Reply:
Looks great!